ambari-common/src/main/python/resource_management/libraries/functions/stack_features.py

```diff
@@ -41,21 +41,22 @@ def check_stack_feature(stack_feature, stack_version):
   :param stack_version: Version of the stack
   :return: Will return True if successful, otherwise, False.
   """
-
-  from resource_management.libraries.functions.default import default
+  from resource_management.libraries.script.script import Script
+  from resource_management.libraries.execution_command.execution_command import ExecutionCommand
   from resource_management.libraries.functions.version import compare_versions
 
-  stack_name = default("/stackSettings/stack_name", None)
+  execution_command = Script.get_execution_command()
+  stack_name = execution_command.get_mpack_name()
   if stack_name is None:
     Logger.warning("Cannot find the stack name in the command. Stack features cannot be loaded")
     return False
-
-  stack_features_setting = stack_settings.get_stack_setting_value(stack_settings.STACK_FEATURES_SETTING)
+  # TODO: If stack_features is needed, this should be added in execution_command lib
+  stack_features_setting = execution_command.get_value("stackSettings/"+stack_settings.STACK_FEATURES_SETTING)
```

Comment:
Yes we should add stack_settings to execution command library.

Reply:
AMBARI-24468 has filed to address stack_settings and cluster_settings